lib/editor/result.js

```diff
@@ -0,0 +1,281 @@
+'use babel'
+import {CompositeDisposable, Emitter} from 'atom'
+import ResultView from './result-view'
+import fastdom from 'fastdom'
+
+let layers = {}
+
+export default class Result {
+  constructor (editor, lineRange, opts) {
+    this.emitter = new Emitter()
+    this.disposables = new CompositeDisposable()
+
+    this.editor = editor
```

Comment:
Hm, we always had it like this: https://github.com/JunoLab/atom-ink/blob/bc2c6adfb5d1fa22bbfbad7fa1dcb3f6593255f7/lib/editor/result.coffee#L36
Can't remember this causing any problems.